client/asset/btc/spv.go

```diff
@@ -408,6 +409,35 @@ func (w *btcSPVWallet) SignTx(tx *wire.MsgTx) error {
 	return txauthor.AddAllInputScripts(tx, prevPkScripts, inputValues, &secretSource{w, w.chainParams})
 }
 
+func (w *btcSPVWallet) SendToOutputs(outputs []*wire.TxOut, keyScope *waddrmgr.KeyScope, account uint32, minConf int32,
+	satPerKb btcutil.Amount, coinSelectionStrategy wallet.CoinSelectionStrategy) (*wire.MsgTx, error) {
+	// Ensure the outputs to be created adhere to the network's consensus rules.
+	for _, output := range outputs {
+		err := txrules.CheckOutput(output, txrules.DefaultRelayFeePerKb)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// Create the transaction and broadcast it to the network. The
+	// transaction will be added to the database in order to ensure that we
+	// continue to re-broadcast the transaction upon restarts until it has
+	// been confirmed.
+	changeTypeOpt := wallet.WithCustomChangeScope(&waddrmgr.KeyScopeBIP0084) // p2wpkh change
```

Comment:
This is why we implement `SendToOutputs`  ourselves now.  btcwallet might make a similar variadic arg change to `SendOutputs`, but we'll still have to override it to match the interface that the ltc and bch packages use.